src/Components/CommentsList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from "react-router-dom";
+import myApi from "./api";
+import { useState } from "react";
+
+export default function CommentsList() {
+  const { article_id } = useParams();
+  const [commentsList, setCommentsList] = useState([]);
+
+  myApi
+    .get(
+      `https://nc-news-6m81.onrender.com/api/articles/${article_id}/comments`
+    )
+    .then((response) => {
+      setCommentsList(response.data);
+    });
+  return (
+    <article>
+      <h1>Comments</h1>
```

Comment:
We might want to add some functionality in case their are no comments associated with an article, once we start letting users delete this could be needed!